pennylane/labs/resource_estimation/ops/op_math/controlled_ops.py

```diff
@@ -252,11 +519,28 @@ class ResourceCCZ(qml.CCZ, re.ResourceOperator):
 
         .. math:: \hat{Z} = \hat{H} \cdot \hat{X} \cdot \hat{H}.
 
-        We replace the Pauli-X gate with a Toffoli gate to obtain our control-control-Z gate.
+        By replacing the :class:`~.ResourceX` gate with a :class:`~.ResourceToffoli` we obtain
+        the controlled decomposition. Specifically, the resources are given by a
+        :class:`~.ResourceToffoli` gate conjugated by a pair of :class:`~.ResourceHadamard` gates.
```

Comment:
Maybe instead of "given by", we try something like "defined as"?

Reply:
replaced